client/source/components/questionsAnswers/QAModule.jsx

```diff
@@ -1,22 +1,53 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect, useContext } from 'react';
+import styled from 'styled-components';
+import ProdContext from '../../ProdContext.js';
 import SearchQuestions from './SearchQuestions';
 import LoadAddQA from './LoadAddQA';
 import DisplayPhotos from './DisplayPhotos';
+import QAList from './QAList';
 import Modal from '../shared/Modal';
 import useModal from '../../useModal';
+import * as requests from '../../utilities/axiosRequests.js';
+import qaData from './exampleDataQA';
 import { Container } from './styles/Containers.styled';
 
 const QAModule = function CreateQAModuleComponent() {
   const { isOpen, onOpen, onClose } = useModal();
+  const { prodID } = useContext(ProdContext);
+  const [questions, setQuestions] = useState(qaData.results);
+  const [questionCount, setQuestionCount] = useState(4);
```

Comment:
I noticed this is being used in your request to get questions to return 4 questions (per BRD). I'm not sure how you were thinking about handling the "More Questions" feature where on click, it should load more questions.

As some ideas, I try to get a bunch of reviews at one time (e.g. 100), though I'm sure there are tradeoffs with this approach, and once a user clicks, I would be able to display more. 

The approach you're heading down here seems you might be thinking when a user clicks, make a new request to the API, and get 2 additional (or however many additional) questions. 

Again, likely trade-offs with both approaches (one approach makes a large request up front, which may impact load time), and the other approach is wait until the user interaction, then make another request to the API.

Reply:
Yes, this was an idea that I thought might be good, but it will probably involve too many API requests. I think I will try an approach to the way I presented the answers in groups of two (after all of the answers were retrieved from the API). Thank you for confirming my suspicion!